### PR TITLE
update friend_id and sqlite seed

### PR DESCRIPTION
**Pull request by arthurtalkgoal**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://arthurtalkgoal-3-0-stable.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://arthurtalkgoal-3-0-stable.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/db/default/spree/states.rb [hunk 1]
```diff
@@ -21,8 +29,15 @@
...
 state_values.call.each_slice(500) do |state_values_batch|
-  connection.execute <<-SQL
-    INSERT INTO spree_states (#{columns})
-    VALUES #{state_values_batch.join(", ")};
-  SQL
-end
+  if connection.adapter_name =~ /SQLite/i
+    connection.execute <<-SQL
+      INSERT INTO spree_states (#{columns})
+      SELECT #{state_values_batch.join(" UNION SELECT ")};
```
Comment (arthurtalkgoal) [inline core/db/default/spree/states.rb]: Hi @Houndci, I don't get this line ***Prefer single-quoted strings inside interpolations.***
can you give an example? thanks much

**Comment by gmacdougall** [comment]
Hi Arthur. Thanks for your work on this. Could you please split this up in to three pull requests:

- Bumping friendly_id version
- Fixing the product spec
- Changes to the county seeds

The first two look fine to be merged, but I'd like to review the country seeds in more detail.

Could you also please provide a description in the pull request of the specific reason that these changes would be helpful?

Thanks.

**Comment by arthurtalkgoal** [comment]
HI @gmacdougall , sure, my pleasure. 

As I don't have much experience in doing multiple pull request on the same branch, can I ask what I can do? (http://stackoverflow.com/questions/8450036/how-to-open-multiple-pull-requests-on-github)

Should I do the pull request and wait for merge one after one?

**Comment by arthurtalkgoal** [comment]
Hi ,

I found out a soln - create branches to compare: http://stackoverflow.com/questions/5256021/send-a-pull-request-on-github-for-only-latest-commit

hope it is the standard ways

```
git checkout -b upstream upstream/master

git cherry-pick <SHA hash of commit>

git push origin upstream

```

**Comment by gmacdougall** [comment]
That's a good way to do it. Feel free to create all the pull requests at once.

**Comment by arthurtalkgoal** [comment]
Ah.... it's hard to PR into spree, it becomes a huge project now I can tell...

@JDutil  just closed my PR since he think it's a duplicate PR of my previous closed one...

**Comment by JDutil** [comment]
I've already merged your previous PR to both 3-0-stable and master so those were duplicates.  All that is left is the sqlite seeds.

**Comment by JDutil** [comment]
Previously merged w/ https://github.com/spree/spree/commit/c22caf798c431f93e5406a078f016a0a403eb95b

**Comment by arthurtalkgoal** [comment]
Oh @JDutil , Thanks for pointing it out. It's my fault don't get it.

**Pull request closed without merging**